docs/dev/ide-setup/eclipse.md

```diff
@@ -0,0 +1,49 @@
+# *Eclipse* Setup
+
+##  Plugins:
+  - ***Buildship Gradle Integration***
+     1. [Install Plugin](https://marketplace.eclipse.org/content/buildship-gradle-integration)
```

Comment:
Haven't used eclipse in a while, but I'm pretty sure the gradle support has been pre-installed for a while now, at least to a degree where the projects is loaded out of the box.

Reply:
Can't rightly speak to this personally. I did have to install this plugin previously and re-installed lately only to get the formatter parts together.

I'd be reluctant to remove it without confirmation. It was a bit painful getting the project to compile in IDEA with the latest java 11 changes, I'm sorry to say I don't have the patience/time/desire to go through a full build in eclipse right now beyond what was needed to get the formatter working.

Reply:
Yeah, java 11 support in ecplise is really not that great.
No objection for leaving it in, I just didn't have to explicitly download this plugin back in the days.